composer.json

```diff
@@ -23,7 +23,7 @@
         "phpstan/phpstan-phpunit": "^0.9.4",
         "phpunit/phpunit": "^7.0",
         "doctrine/coding-standard": "^4.0",
-        "doctrine/common": "^2.8",
+        "doctrine/common": "^2.9@dev",
```

Comment:
Urgh, this is still required? (I missed that before)

Reply:
@Ocramius yes, because of this https://github.com/doctrine/reflection/blob/master/lib/Doctrine/Common/Reflection/RuntimePublicReflectionProperty.php#L25

Which is not a standalone project. We could possibly remove the tests that verify that functionality and we could temporarily remove the doctrine/common dependency for the tests. Then reintroduce the tests once doctrine/common 2.9 stable is released. It just buys is a few less steps I think.

Reply:
Also OK to have a release with a failure as long as we know what we are doing